### PR TITLE
Add missing files, update metadata post-migration

### LINKED ISSUE #2: Update metadata.json to reflect transfer to Vox Pupuli
The metadata is currently pointing to my account instead of the voxpupuli org.

### PR DESCRIPTION
**Pull request by genebean**
#### Pull Request (PR) description

This PR adds a couple of files that inadvertantly got left out of #1. It also updates the metadata.json to reflect that this module has been moved to Vox Pupuli.

#### This Pull Request (PR) fixes the following issues

Fixes #2

### CODE REVIEW

**Comment by genebean** [comment]
@bastelfreak or other admin - will you fix the branch protection rule... I made it target the wrong test (it should be `CI / Puppet / Test suite (pull_request)`

**Comment by bastelfreak** [comment]
sadly the current release workflow will not work with branch protection enabled at the moment.  I disabled it.

**Review by bastelfreak — APPROVED** [review]

File: metadata.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "genebean-falco",
+  "name": "puppet-falco",
   "version": "0.4.0",
-  "author": "genebean",
+  "author": "puppet",
```
Comment (alexjfisher) [inline metadata.json:4]: Is this right?  I don't think it's just supposed to be a copy of the module's namespace.

**Comment by genebean** [comment]
Thanks @bastelfreak

File: metadata.json (hunk 1 above)
Comment (bastelfreak) [reply metadata.json:4]: @genebean @alexjfisher is right. the author needs to be `Vox Pupuli`. Can you please update it?

Comment (genebean) [reply metadata.json:4]: I know some tooling expects it to be the forge username... I can't remember if that is PDK specific or something else though to be honest.

**Pull request merged**